wavenet.py

```diff
@@ -102,7 +106,7 @@ def _create_network(self, input_batch):
 
         return conv2
 
-    def loss(self, input_batch, name='wavenet'):
+    def loss(self, input_batch, add_summary=True, name='wavenet'):
```

Comment:
I would expect that `add_summary` would also turn all other summaries on/off.
Let's leave this change out of this PR for now, then we can make all summaries configurable at a later point.